R/tm_g_association.R

```diff
@@ -48,6 +48,8 @@
 #' To learn more please refer to the vignette
 #' `vignette("transform-module-output", package = "teal")` or the [`teal::teal_transform_module()`] documentation.
 #'
+#' @inheritSection teal::example_module Reporting
```

Comment:
@averissimo I get a warning that this section does not exist. I was checking it's [companion teal PR](https://github.com/insightsengineering/teal/pull/1541/files) where I would have expected this section and it was not there. Am I missing something or do we need to updated the `teal::example_module` doc?

Reply:
I confirm. Section probably got removed here and `teal.modules.*` can't inherit reporter documentation. I think it is worth to have a `# Reporting` (and maybe `@return`) inherited from `teal::example_module` (or from other place in teal). Name of the section could be more universal as SRC is also a concern

Reply:
I was trying to track down where this cames from and finally found it in an open PR: https://github.com/insightsengineering/teal/pull/1545/files#diff-30b1b14a6fc7f2b6907f241d0f5d2e780dea7396ded16caffc8147b3d0fdf5a7

The `@return` tag will be tricky as these functions return the module itself. We could extend it to include what the `srv` returns

Reply:
Maybe something like this?
> #' @return `teal_module` whose `server` returns `reactive(<teal_data>)` ...